torch/nn/functional.py

```diff
@@ -3764,7 +3764,7 @@ def multi_head_attention_forward(query,                           # type: Tensor
     assert key.size() == value.size()
 
     head_dim = embed_dim // num_heads
-    assert head_dim * num_heads == embed_dim, "embed_dim must be divisible by num_heads"
+    assert head_dim * num_heads == embed_dim, f"embed_dim = {embed_dim} must be divisible by num_heads = {num_heads}"
```

Comment:
I don't think `f` strings are valid strings in Python 2, and we haven't fully dropped support for Python 2 yet (though that is coming soon: https://github.com/pytorch/pytorch/issues/23795)